.github/workflows/push-to-prod-draft.yml

```diff
@@ -74,13 +46,13 @@ jobs:
         if: matrix.sbc == 'rockpi' && success()
         with:
           balena_api_token: ${{secrets.BALENA_API_TOKEN}}
-          balena_command: "deploy nebraltd/helium-${{ matrix.variant }}-${{ matrix.frequency }}${{ env.ROCKPI }} --logs --debug --nocache --build --draft"
+          balena_command: "deploy nebraltd/helium${{ env.ROCKPI }} --logs --debug --nocache --build --draft"
```

Comment:
Can you remind me where env.ROCKPI is being set and what values it takes on?

Reply:
https://github.com/NebraLtd/helium-miner-software/blob/81fb2ef903e6c44df6eb9945a313b829fbf658c1/.github/workflows/push-to-prod-draft.yml#L28